src/components/AdminLink.js

```diff
@@ -0,0 +1,26 @@
+/* @flow */
+
+import type { RouteParams } from "../types";
+
+import React, { Component } from "react";
+import { Link } from "react-router";
+
+import url from "../url";
+
+
+type Props = {
+  name: string,
+  params: RouteParams,
+  children: any,
+};
+
+export default class AdminLink extends Component {
```

Comment:
Haha famous last words. Basically to create an `AdminLink` (which wraps a react-router `Link`) in a test environment, you basically have to setup the entire react-router component, as `Link` needs a router context to create the appropriate `href` we want to test.

I abandonned the idea, flow types will do for now.